packages/app/src/app/components/CreateNewSandbox/CreateSandbox/elements.ts

```diff
@@ -38,7 +37,7 @@ export const ModalBody = styled.div`
 `;
 
 export const ModalSidebar = styled.div`
-  width: 224px;
+  width: 176px;
```

Comment:
note: box-sizing is content-box for most of the app, so padding adds to the size, a bit annoying if you ask me but changing this now across the app would be suicide :D